src/components/DeckMechanisms/handleCardEffect.js

```diff
@@ -153,7 +155,7 @@ const handleCardEffect = (state, card, mode) => {
 
       if (mode !== 'MANUAL' && hand.length < 4) {
         draw(state)
-        if (card.level >= 4 && hand.length < 3) draw(state)
+        if (card.level >= 4 && hand.length < 4) draw(state)
```

Comment:
What happened here? Was it wrong?

Reply:
With 3 cards in hand, playing Freebooters lvl 4/5 would draw only one card:
+ Freebooters played (2 cards left)
+ First draw (3 cards in hand)
+ Test fails

Reply:
Should have made a different PR for this though